valorant/client.py

```diff
@@ -284,17 +293,17 @@ def get_user(self, puuid: t.Text) -> t.Optional[AccountDTO]:
         """
 
         try:
-            r = self.handle.call("GET", "puuid", puuid=puuid)
+            r = self.handle.call("GET", "puuid", route=True, puuid=puuid)
         except requests.exceptions.HTTPError as e:
-            if e.response.status_code == 400:
+            if e.response.status_code in [400, 404]:
```

Comment:
When using `in` to check if a variable is within a set of possibilities, it is generally better to use _tuples_ instead of _lists_, since they are

- Faster
- More memory efficient
- Immutable

```suggestion
            if e.response.status_code in (400, 404):
```

Reply:
Noted! 👍🏽 I'll get back to this when I can. Thanks for the tip.